### PR TITLE
Updated src,naming conventions,comments.Please review.

### PR DESCRIPTION
**Pull request by NitinSaxenait**
Please use this PR template
What does this change do?
feature added of complex number and student operations
Any additional information for the reviewer to start
NA

How should this be manually tested?
We need window OS in which Rust is installed. then execute this rust program

Which Trello task and sub-task does this change relate to?

Are there any changes pending?
No

Does any team have to be notified of changes in this feature?
No, only for the mentor.

Definition of Done:

Is there >80% unit test code coverage?
if not please mention the Code Coverage of code.

code coverage:
src/complex_number.rs 14/14
src/student_score.rs 16/34
src/test.rs: 29/29
76.62% coverage , 59/77

Does this PR add new dependencies? If so, please list out the same.
No

Will this feature require a new piece of infrastructure to be implemented?
Is there appropriate logging included?
yes
Does the project compile ok?
yes

Have Clippy violations been fixed?
yes

Have Code is properly formatted?
yes

### CODE REVIEW

File: Assignments/assignment5/src/complex_number.rs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+// function addition performing addition operation on complex numbers.
+//
+// #Arguemnts
+//
+// &self:=> taking variants of structure ComplexNumber.
+//
+// Return
+//
+// return String add_real_values and add_imaginary_value.
+impl ComplexNumbers {
```
Comment (Dheerajjnv) [inline Assignments/assignment5/src/complex_number.rs:21]: Return self instead of string.

File: Assignments/assignment5/src/student_score.rs [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    pub department: String,
+    pub school: String,
+}
+// structure Score containing score of different subjects.
+//
+// #Variants
+//
+// hindi, english, maths, science
+pub struct Score {
+    pub(crate) hindi: i32,
```
Comment (Dheerajjnv) [inline Assignments/assignment5/src/student_score.rs:19]: Remove crate

File: Assignments/assignment5/src/student_score.rs [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+            / 4) as f32;
+        average_of_marks
+    }
+    // This function is used to compare student one marks with other other student marks
+    //
+    // #Arugments
+    //
+    // self and second student
+    //
+    pub fn compare_student(&self, second_student: &Score) {
```
Comment (Dheerajjnv) [inline Assignments/assignment5/src/student_score.rs:99]: Pass the Other student instead of the score.

**Review by Dheerajjnv — COMMENTED** [review]
Remove the deleted file as it already pushed.

File: Assignments/assignment5/src/student_score.rs [hunk 4]
```diff
@@ -0,0 +1,117 @@
+// structure StudentInformation containing details of students.
+//
+// #Variants
+//
+// name, roll_no, score_of_each_subject, department, school.
+pub struct StudentInformation {
+    pub name: String,
+    pub roll_no: u32,
+    pub(crate) score_of_each_subject: Score,
```
Comment (Dheerajjnv) [inline Assignments/assignment5/src/student_score.rs:9]: No need of crate.

File: Assignments/assignment5/src/test.rs [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+// test cases for pass_student
+#[test]
+fn test_pass_student() {
+    let student1_marks = Score {
+        hindi: 32,
+        english: 31,
+        maths: 74,
+        science: 55,
+    };
+    let mut pass_check_test = StudentInformation::new(&student1_marks);
```
Comment (Dheerajjnv) [inline Assignments/assignment5/src/test.rs:89]: Pass student as a parameter

**Review by Dheerajjnv — CHANGES_REQUESTED** [review]
Remove Deleted file.

**Review by sonukumar-a11 — APPROVED** [review]
please remove the unwanted file in the git

**Pull request closed without merging**